app/lists.rb

```diff
@@ -0,0 +1,43 @@
+module Listing
+  extend Discordrb::Commands::CommandContainer
+  
+  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	wing_list = Array.new
```

Comment:
This isn't a global variable. You won't be able to initialize it like this, nor is there any need to.